tests/consensus_test.rs

```diff
@@ -231,15 +236,15 @@ impl NodeForTest {
     }
 
     pub async fn total_messages(&self) -> usize {
-        let messages = self
-            .block_store
-            .get_blocks(0, None, &PageOptions::default())
-            .unwrap()
-            .blocks
-            .into_iter()
-            .map(|b| b.shard_chunks[0].transactions[0].user_messages.len());
-
-        messages.len()
```

Comment:
This was not actually counting the messages. And no messages were included in the blocks because the events didn't exist.